templates/valkey-try-me.html

```diff
@@ -0,0 +1,277 @@
+{% extends "fullwidth.html" %}
+{%- block head -%}
+    <title>Try Valkey</title>
+
+    <!-- Scripts -->
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/v86/libv86.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/xterm/xterm.min.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/pako/pako.min.js"></script>
+    <script src="https://dtq9kvai1345o.cloudfront.net/vos/v86/serial_xterm.js"></script>
+    
+    <!-- Styles -->
+    <link rel="stylesheet" href="https://dtq9kvai1345o.cloudfront.net/vos/xterm/xterm.css" />
+
+    <style>
```

Comment:
Let's split this off in to a dedicated CSS file. I'm told embedded CSS is worse for performance.

Reply:
I am not sure I understand the idea - can explain which part should we split?